clj-refactor.el

```diff
@@ -2059,17 +2121,24 @@ will add the corresponding require statement to the ns form."
                             (not (cljr--in-number-p))
                             (clojure-find-ns)
                             (cljr--unresolved-alias-ref (cljr--ns-alias-at-point))))
-    (when-let (aliases (cljr--magic-requires-lookup-alias alias-ref))
-      (let ((short (cl-first aliases))
-            ;; Ensure it's a list (and not a vector):
-            (candidates (mapcar 'identity (cl-second aliases))))
-        (when-let (long (cljr--prompt-user-for "Require " candidates))
-          (when (and (not (cljr--in-namespace-declaration-p (concat ":as " short "\b")))
-                     (not (cljr--in-namespace-declaration-p (concat ":as-alias " short "\b")))
-                     (or (not (eq :prompt cljr-magic-requires))
-                         (not (> (length candidates) 1)) ; already prompted
-                         (yes-or-no-p (format "Add %s :as %s to requires?" long short))))
-            (cljr--insert-require-libspec (format "[%s :as %s]" long short))))))))
+    (if cljr-slash-uses-suggest-libspec
+        (when-let (libspec
```

Comment:
Maybe add a `;; comment` explaining the case where `libspec` is nil

Reply:
I think this will be easier to follow once the `cljr-slash-uses-suggest-libspec` feature flag is gone. If we can't find a libspec we don't suggest or insert one, which is the same behavior we used previously but required two `when-let`. We could extract the case where `cljr-slash-uses-suggest-libspec` is nil into it's own function, maybe that would be clearer? Could you elaborate on what your concern is here, I don't think I understand.

Reply:
Couldn't think of a name to keep this clear, so just left a comment. Formatting was off anyway.